CHANGELOG.md

```diff
@@ -12,6 +12,12 @@
 
 ### Dependency updates
 
+## [1.5.0] 2021-08-03
```

Comment:
Please adhere to the changelog rules, usually we create a separate branch when making a release: https://github.com/teamleadercrm/ui-icons#how-to-make-a-release

Reply:
I will remove this and add another branch for the release.